src/utils/validation.ts

```diff
@@ -30,8 +30,8 @@ export const beaconSchema = z.object({
   updateConditionPercentage: z.number(),
   chains: z.array(extendedChainDescriptionSchema),
   signedKeeperConditions: z.object({
-    deviationFactorThreshold: z.number(),
-    ttlMinutes: z.number(),
+    updateConditionPercentage: z.number(),
+    ttlSeconds: z.number(),
```

Comment:
This should be updated based on the README.
Why do you have separate types and not use types inferred by zod from the schema itself?

Reply:
I'm not hugely familiar with zod, I didn't realise the schema could be used like this.
Can we get this PR merged and then we can merge the types/schema in a separate PR?
I will make make the changes you've highlighted now (thanks :relaxed: ).

Reply:
Of course :slightly_smiling_face:

Reply:
Furtther to the types vs schema, I followed the implementation example of Airnode and Airkeeper, which both have separate types and schema, but this may be due to legacy and/or cargo-culting.

Reply:
The Airnode one is yet to be updated, that's true but the Airkeeper one is using them. It just might seem it's not because we're merging two configuration files there. But [all of these](https://github.com/api3dao/airkeeper/blob/main/src/types.ts#L25-L28) for example are imported from the schema.

Reply:
ah I see, okay, that's pretty easy to change, I'll that as part of this PR. Thanks for pointing that out.